packages/components/src/mobile/bottom-sheet/bottom-sheet-navigation/navigation-screen.native.js

```diff
@@ -79,11 +80,19 @@ const BottomSheetNavigationScreen = ( {
 	};
 	return useMemo( () => {
 		return isScrollable || isNested ? (
-			<View onLayout={ onLayout }>{ children }</View>
+			<View
+				onLayout={ onLayout }
+				testID={ `navigation-screen-${ name }` }
```

Comment:
As this is an element with no user-facing identifier, a `testID` was required to query the element.